package.json

```diff
@@ -23,7 +23,9 @@
     "start:background": "npm run cli -- background",
     "pretest": "npm run lint && cross-env NODE_ENV=test npm run db:migrate:reset",
     "test": "jest --testEnvironment=node --runInBand",
-    "typeorm": "typeorm-ts-node-commonjs"
+    "typeorm": "typeorm-ts-node-commonjs",
+    "temporal:dev": "cross-env TEMPORAL_ADDRESS=0.0.0.0:7233 nodemon ./src/temporal/local.ts",
+    "temporal:server": "temporal server start-dev --ip=0.0.0.0"
```

Comment:
does it require anything to be installed locally?

Reply:
Yes, you will have to install `temporal` while we don't have it within our Tilt ecosystem. I might update the README, once we get merge.